django_react_proj/users-fe/src/App.js

```diff
@@ -24,6 +24,8 @@ const App = () => {
   };
   const [currentDate, setCurrentDate] = useState(new Date());
   const [workHours, setworkHours] = useState(8);
+  const [firstNotifiaction, setFisrtNotification] = useState(15);
```

Comment:
Optionally instead of passing state and setters through multiple components you can use single context

https://pl.reactjs.org/docs/context.html

You can use it as `SettingsProvider` where you keep all state / setters settings component needs to be accessed by other components:

```js
const settingsContext = React.createContext()
function SettingsProvider({ children }) {
  const [cos, setCos] = useState(...)
  const [cos1, setCos1] = useState(...)
  
  const updateCos1 = (value) => setCos1(value)
  
  const value = {
    cos,
    cos1
    updateCos1
  }

  return <settingsContext.Provider value={value}>{children}</settingsContext.Provider>
}
```

Then you can use `useContext` to access `value` from any component (those components have to be wrapped with the same `SettingsProvider`)

Not tested but you should end up in something similar.